tests/test_diaForcedSource.py

```diff
@@ -59,6 +59,13 @@ def create_test_dia_objects(n_points, wcs, startPos=100):
         src['id'] = src_idx
         src.setCoord(wcs.pixelToSky(startPos + src_idx,
                                     startPos + src_idx))
+
+    # Add one point outside of the CCD.
+    src = objects.addNew()
+    src['id'] = 10000000
+    src.setCoord(wcs.pixelToSky(-100000,
+                                -100000))
```

Comment:
Actually, one more suggestion: consider testing points that are out-of-bounds in only X or only Y, or some that are *just* out-of-bounds (especially points that would be treated differently by `Box2I` vs. `Box2D`).